CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+### Added
+
+### Changed
+
+### Deprecated 
+
```

Comment:
Why add these? We can add them after the release, so we can have the v0.9.0 as the first header.

Reply:
removed